tests/backup/backup_delete_test.go

```diff
@@ -394,8 +396,38 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
 				for _, vol := range bkpInspectResponse.GetBackup().GetVolumes() {
 					backupId := vol.GetBackupId()
 					log.InfoD(fmt.Sprintf("Backup Name: %s; BackupID: %s", incrementalBackupName, backupId))
-					dash.VerifyFatal(strings.Contains(backupId, "incr"), true,
-						fmt.Sprintf("Check if the backup %s is incremental or not ", incrementalBackupName))
+					if strings.Contains(backupId, "incr") {
+						dash.VerifyFatal(strings.Contains(backupId, "incr"), true,
+							fmt.Sprintf("Check if the backup %s is incremental or not ", incrementalBackupName))
+					} else {
+						// Attempting to take backups and checking if they are incremental or not
+						log.InfoD("New backup wasn't an incremental backup hence recreating new backup")
+						for i := 0; i < 3; i++ {
```

Comment:
Can you declare a variable like `maxBackupsBeforeIncremental = 3` at the top or provide a comment as to why we are looping only thrice so that we dont forget why we did this when we look back in hindsight?